ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -104,3 +108,29 @@ def nanpercentile(
                     arrayofpercentiles.append(_cpercentile(ii, i))
                 resultarray.append(arrayofpercentiles)
         return resultarray
+
+
+
+@ivy.numpy_frontend
+@ivy.handle_numpy_out
+@ivy.handle_numpy_dtype
+@ivy.to_ivy_arrays_and_back
+@ivy.handle_numpy_casting
+@ivy.from_zero_dim_arrays_to_scalar
+def quantile(
+    a,
+    q,
+    /,
+    *,
+    axis=None,
+    out=None,
+    overwrite_input=False,
+    method="linear",
+    keepdims=False,
+    interpolation=None,
+):
+
+    ret = ivy.quantile(
+        a, q, axis=axis, out=out, overwrite_input=overwrite_input, method=method, keepdims=keepdims, interpolation=interpolation
+    )

```

Comment:
Whilst I understand that keyword arguments can be in any order, it is often a good practice to keep it in the order specified by the function declarator which you can see in the documentations I linked before or in the github repository here [here](https://github.com/unifyai/ivy/blob/master/ivy/functional/ivy/experimental/statistical.py).  Please note that `overwrite_input` isn't an argument that exists for ivy.quantile.

Also I think studying at least the numpy backend [here](https://github.com/unifyai/ivy/blob/master/ivy/functional/backends/numpy/experimental/statistical.py) for ivy.quantile will help in understanding how the `interpolation` keyword should be assigned